requirements-common.txt

```diff
@@ -23,3 +23,4 @@ typing_extensions
 filelock >= 3.10.4 # filelock starts to support `mode` argument from 3.10.4
 pyzmq
 gguf == 0.9.1
+compressed-tensors == 0.5.0
```

Comment:
We should check whether this breaks non cuda platforms

Reply:
compressed-tensors has pretty lenient deps, but maybe requiring torch (even if not a specific version) could mess things up?
```
def _setup_install_requires() -> List:
    return ["torch>=1.7.0", "transformers", "accelerate", "pydantic>=2.0"]
```

Reply:
requirements-common.txt [requires torch >= 2.1.0](https://github.com/vllm-project/vllm/blob/main/requirements-common.txt#L21C1-L21C52), so this should be okay for the common case?